docs/models/examples.md

```diff
@@ -1,15 +1,17 @@
 # Larq Zoo Examples
 
-## Classify ImageNet classes with Bi-Real Net
+## Classify ImageNet classes with BinaryDenseNet-45
+
+Before running the example below, download [this picture](https://raw.githubusercontent.com/larq/zoo/master/tests/fixtures/elephant.jpg) of an elephant to test your model.
 
 ```python
 import tensorflow as tf
 import numpy as np
 import larq_zoo as lqz
 
-model = lqz.BiRealNet(weights="imagenet")
+model = lqz.BinaryDenseNet45(weights="imagenet")
 
-img_path = "tests/fixtures/elephant.jpg"
+img_path = "path/to/elephant.jpg"
```

Comment:
If the example is run from the repo directory, it will find the image. But I'd also be OK to change this.

Reply:
Would have to be the `larq-zoo` directory even though this documentation now lives in `larq`, and if people read this from the website I would also say this is clearer.

Reply:
OK, makes sense